colossalai/shardformer/shard/shard_config.py

```diff
@@ -49,6 +49,7 @@ class ShardConfig:
     make_vocab_size_divisible_by: int = 64
     gradient_checkpoint_config: Optional[GradientCheckpointConfig] = None
     extra_kwargs: Dict[str, Any] = field(default_factory=dict)
+    use_zbv: bool = False
```

Comment:
this can be got from pipeline_stage_manager

Reply:
> this can be got from pipeline_stage_manager

Replaced to pipeline_stage_manager.use_zbv;